File: Back End/C/Chapter 3 - Control Statement/2K221010/patteren_alphabet1_ascending.c
//Loops:

//Pattern Programs

/*

Ex. Alphabet Pattern 1 - Ascending

A
A B
A B C
A B C D
A B C D E

*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

    for(i=1;i<=count;i++){
        for(j=1;j<=i;j++){
            printf("%c",j+64);
        }
        printf("\n");
    }
}

File: Back End/Python/Chapter 5 - Control Statements/2K221229/patteren_alphabet1_ascending.py
# //Loops:
#
# //Pattern Programs
#
# /*
#
# Ex. Alphabet Pattern 1 - Ascending
#
# A
# A B
# A B C
# A B C D
# A B C D E
#
# */

limit = int(input("Enter the limit: "))

for i in range(0, limit + 1, 1):
    for j in range(0, i, 1):
        print(chr(j + 65), " ", end='')
    print(end='\n')

# print(chr(65))
